src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -495,15 +492,10 @@ private void setDefaultValues() {
 
     checkBoxTabulateByPrecinct.setSelected(ContestConfig.SUGGESTED_TABULATE_BY_PRECINCT);
     checkBoxGenerateCdfJson.setSelected(ContestConfig.SUGGESTED_GENERATE_CDF_JSON);
-    checkBoxSequentialMultiSeat.setSelected(ContestConfig.SUGGESTED_SEQUENTIAL_MULTI_SEAT);
```

Comment:
Do we want to suggest "standard" for `winnerElectionMode`?

Reply:
Yeah, I had it that way before. I ran into an annoying edge case with that, but I'll try again to fix it, and describe it to you if I'm still stuck.